pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -3331,23 +3331,13 @@ CREATE UNIQUE INDEX ON public.t_118626(c) WHERE n>= 1;
 statement ok
 ALTER TABLE public.t_118626 ALTER PRIMARY KEY USING COLUMNS(b);
 
-subtest end
-
-subtest add_default_empty_array
```

Comment:
this is removing an existing test case. instead, add the new test without removing the old one.